spec/spec_helper.rb

```diff
@@ -57,6 +57,15 @@
   config.mock_with :rspec do |mocks|
     mocks.yield_receiver_to_any_instance_implementation_blocks = true
   end
+
+  # Monkey-patch Marshal.dump to enable dumping of mocked or stubbed objects. By
+  # default this will not work since RSpec mocks works by adding singleton methods
+  # that cannot be serialized. This patch removes these singleton methods before
+  # serialization. Potentially but unlikely to be removed post upgrade.
+  config.mock_with :rspec do |mocks|
+    mocks.patch_marshal_to_support_partial_doubles = true
```

Comment:
[Yes it is](https://github.com/rspec/rspec-mocks/blob/master/lib/rspec/mocks/configuration.rb#L174-L187)

I'd like to see if I can remove this later and it would be easier just to upgrade and figure it out then (But given our extensive use of that lib, maybe not a big deal to leave it either ¯\\_(ツ)_/¯)